src/main/java/org/sagebionetworks/web/client/place/users/RegisterAccount.java

```diff
@@ -1,19 +1,17 @@
 package org.sagebionetworks.web.client.place.users;
 
-import com.google.gwt.place.shared.Place;
 import com.google.gwt.place.shared.PlaceTokenizer;
 import com.google.gwt.place.shared.Prefix;
+import org.sagebionetworks.web.client.place.ParameterizedPlace;
 
-public class RegisterAccount extends Place {
+public class RegisterAccount extends ParameterizedPlace {
 
-  private String token;
+  public static final String EMAIL_QUERY_PARAM = "email";
+  public static final String MEMBERSHIP_INVTN_QUERY_PARAM =
+    "membershipInvtnSignedToken";
 
   public RegisterAccount(String token) {
```

Comment:
Note that there are cases in the codebase that point to RegisterAccount with a default token ("0").  I tested that the presenter just redirects to the OneSage registration page without any additional query params (besides the appId used to brand the One Sage site)